life_expectancy/tests/test_main.py

```diff
@@ -0,0 +1,14 @@
+"""Tests for the main module"""
+
+import pandas as pd
+
+from life_expectancy.main import main
+
+
+def test_main(pt_life_expectancy_expected):
+    """Run the `main` function and compare the output to the expected output"""
+    pt_life_expectancy_actual = main('PT').reset_index(drop=True)
```

Comment:
I would prefer to mock the to_csv function again in order to avoid making any changes to the files that are used by the application.

Reply:
`with patch('life_expectancy.cleaning.save_data', side_effect=print("Saved to csv!!!")):
     pt_life_expectancy_actual = main('PT').reset_index(drop=True)
     (...)
`

Reply:
Resolved in this commit c020409ab73b440541aa1bf2de71646cdbf9fbdd.